server/statisticalFactsheets/statisticalFactsheetsRepository.js

```diff
@@ -3,14 +3,31 @@ const db = require('../db/db')
 
 const getStatisticalFactsheetTableAgg = async (schemaName) => {
   const result = await db.query(`
-  SELECT
-    *
-  FROM
-    ${schemaName}.statistical_factisheets_table_agg
-  ORDER BY
-    level ASC
+  SELECT *
+  FROM ${schemaName}.statistical_factsheets_view
+  -- All the rows/variables used in statistical factsheet 
+  WHERE row_name = 'Carbon stock in biomass (Gt)'
```

Comment:
do you need where conditions here? doesn't the statistical_factsheets_view contains only the rows needed?